src/pharmpy/tools/amd/run.py

```diff
@@ -172,25 +159,15 @@ def run_amd(
 
     from pharmpy.model.external import nonmem  # FIXME: We should not depend on NONMEM
 
-    if modeltype == 'pkpd':
-        dv = 2
-        iiv_strategy = 'pd_fullblock'
-        # FIXME : DO THIS ONLY ONCE (DUPLICATED FURTHER DOWN)
+    if search_space is not None:
         try:
-            input_search_space_features = [] if search_space is None else mfl_parse(search_space)
+            ss_mfl = mfl_parse(search_space, True)
```

Comment:
Since the MFL object class has not yet been fully integrated everywhere, this is to keep its default behavior to be the same as it used to be. With this update in place, its previous behavior should perhaps be revised, however.